time-of-day test failure in testing rfc-index sync

```
======================================================================
FAIL: test_rfc_index (ietf.sync.tests.RFCSyncTests.test_rfc_index)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/datatracker/datatracker/ietf/sync/tests.py", line 402, in test_rfc_index
    self.assertIn(f"added RFC published event at {rfc_events[0].time:%Y-%m-%d}", rfc_events[0].desc)
AssertionError: 'added RFC published event at 2023-11-22' not found in "Received changes through RFC Editor sync (created document RFC 1234, created became rfc relationship between draft-bogusperson-acronym7239-musings3135 and RFC 1234, set title to 'A Testing RFC', set abstract to 'This is some interesting text.', set pages to 42, set standardization level to Proposed Standard, added RFC published event at 2023-11-21, created updates relation between RFC 1234 and RFC 123, added Errata tag)"

----------------------------------------------------------------------
```

--- a/ietf/sync/tests.py
+++ b/ietf/sync/tests.py
@@ -399,7 +399,7 @@ class RFCSyncTests(TestCase):
                 self.assertIn("set abstract to 'This is some interesting text.'", rfc_events[0].desc)
                 self.assertIn("set pages to 42", rfc_events[0].desc)
                 self.assertIn("set standardization level to Proposed Standard", rfc_events[0].desc)
-                self.assertIn(f"added RFC published event at {rfc_events[0].time:%Y-%m-%d}", rfc_events[0].desc)
+                self.assertIn(f"added RFC published event at {rfc_events[0].time.astimezone(RPC_TZINFO):%Y-%m-%d}", rfc_events[0].desc)
                 self.assertIn("created updates relation between RFC 1234 and RFC 123", rfc_events[0].desc)
                 self.assertIn("added Errata tag", rfc_events[0].desc)
             else:

